generic/sysbench.py

```diff
@@ -67,6 +68,11 @@ def setUp(self):
                 self.bch = self.params.get('branch', default='master')
                 git.get_repo(self.urllink, branch=self.bch,
                              destination_dir=self.teststmpdir)
+                if self.urllink == 'https://github.com/akopytov/sysbench.git':
```

Comment:
Instead of  urllink check can you check for ppc as the fix is only for ppc. 
if "ppc"  in distro.detect().arch:
then download the luajit for ppc

Reply:
@narasimhan-v  not sure it temporary as akopytov is working on . And this need for ppc system in as x86 no issue as `vm_ppc64.dasc` broken in ppc arch , so i adopted hari's suggestion

Reply:
@harihare i think we need to have both condition as if any other url passed from ymal file it will broke